numpy/core/_add_newdocs.py

```diff
@@ -3770,20 +3770,25 @@ def luf(lamdaexpr, *args, **kwargs):
     ----------
     file : str
         A string naming the dump file.
+    protocol : int, optional
+        The pickle protocol used to serialize the array.
+        The default is 2 (in order to maintain python 2 support).
 
     """))
 
 
 add_newdoc('numpy.core.multiarray', 'ndarray', ('dumps',
     """
-    a.dumps()
+    a.dumps(protocol=2)
 
     Returns the pickle of the array as a string.
     pickle.loads or numpy.loads will convert the string back to an array.
 
     Parameters
     ----------
-    None
+    protocol : int, optional
+        The pickle protocol used to serialize the array.
+        The default is 2 (in order to maintain python 2 support).
```

Comment:
You probably need to use the `.. versionadded:: 1.16.0`  directive:



```
    protocol : int, optional
        The pickle protocol used to serialize the array.
        The default is 2 (in order to maintain python 2 support).

         .. versionadded:: 1.16.0
```

Reply:
done!